### ISSUE TITLE
Is this project still being worked on?

### ISSUE THREAD

**Issue by Jasontyler221** [AeonLucid/Prospect#2]
Hey there, tried the game recently, had a terrible time despite the game being fairly fun, was wondering if this project is still something you work on when bored or if it no longer gets attention

**Comment by AeonLucid** [comment]
Not really any reason to, the game is out now and not as fun anymore. Might reuse later for another UE4 game.

**Comment by Jasontyler221** [comment]
I see, thanks for letting me know

**Comment by jarrodnorwell** [comment]
@Jasontyler221 did you happen to play the closed beta 2? I want to try revive this project in some way but I started playing in the official release

**Comment by Typhon999** [comment]
I don't think so. I was playing long before the first official season
began, but back then I didn't have the ability to invest the adequate time
required to really learn the dynamic of the game. I went through 4 or 5
resets before the first official "season" was announced. Only really began
to compete there during the 2 expansions before the end.
I figured the community would end up finding a way to build a free shard or
perhaps integrate some way to host private/dedicated servers like valheim
or DayZ, etc. but it seems the backend security is more of a pain in the
ass than it's worth to most.
Is there something you are needing from the beta 2 client? I'm sure there's
a copy floating around somewhere I could find
